src/kernels/Conv_Winograd_v21_1_2_f3x2_fp32_stride1_group.s

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (c) 2021 Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+.include "Conv_Winograd_v21_1_2_metadata.inc"
+
+KERNEL_PROLOG f3x2_fp32_stride1_group
+
+.if (.amdgcn.gfx_generation_number == 9)
+    .if (.amdgcn.gfx_generation_stepping == 10)
+        .error "gfx90a is not supported yet"
+    .else
+        .include "Conv_Winograd_v21_1_2_f3x2_gfx9_fp32_stride1_group.inc"
```

Comment:
Please make file naming consistent with kernel names. Code in conv_winoRxS_f3x2.cpp might also be simplified if the kernel name and file name would be consistent.

Reply:
@Kirpich30000 You review does not block merging. Is it wnat you want, i.e. is it Ok to merge this without resolving your review comments first?

Reply:
I preffer to resolve the issues.